build.gradle

```diff
@@ -208,6 +217,148 @@ testClusters.integTest {
     systemProperty("java.library.path", "$rootDir/jni/release")
 }
 
+String bwcVersion = "1.0.0"
+String baseName = "knnBwcCluster"
+String bwcFilePath = "src/test/resources/org/opensearch/knn/bwc/"
+
```

Comment:
Need some descriptive comments here explaining how this works.

Reply:
Added more comments